modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/am/integration/tests/streamingapis/serversentevents/ServerSentEventsAPITestCase.java

```diff
@@ -234,6 +236,17 @@ public void testInvokeSseApi() throws Exception {
         sseReceiver.setReceivedDataEventsCount(0);
     }
 
+    @Test(description = "Invoke SSE API with invalid token", dependsOnMethods = "testInvokeSseApi")
+    public void testInvokeSseApiWithInvalidToken() throws Exception {
+        invokeSseApi("00000000-0000-0000-0000-000000000000", 30000);
+        int received = sseReceiver.getReceivedDataEventsCount();
+        Assert.assertEquals(received, 0);
```

Comment:
let's print a message here if it fails with received count ?

Reply:
Done via https://github.com/wso2/product-apim/pull/10992/commits/99f4805af2b1b07ed841e5e5bd000c5178af5384